### PR TITLE
fix(php8): non-capturing exception catches

### LINKED ISSUE #1760: PHP8: catch block without variable declaration
**Prettier 2.3.1**

**PHP Plugin 0.17.1**

**Input:**

```php
try {
    // Something
} catch (\Exception) {
    // Nothing
}
```

**Expected:**
Valid PHP 8 code; file should be formatted.

**Result:**

```
Parse Error : syntax error, unexpected ')' on line 13
  11 | try {
  12 |     // Something
> 13 | } catch (\Exception) {
     |                   ^
  14 |     // Nothing
  15 | }
```

### PR DESCRIPTION
**Pull request by czosel**
Fixes #1760

### CODE REVIEW

**Comment by czosel** [comment]
@cseufert Could you look over this?

File: tests/errors/errors.php [hunk 1]
```diff
@@ -46,3 +46,16 @@
 } finally {
     // Comment
 }
+
+// PHP 8.0 non-capturing exception catch
+try {
+    // Something
+} catch (\Exception) {
+    // Nothing
+}
```
Comment (cseufert) [inline tests/errors/errors.php:55]: Is it worth making the example something that will actually be reformatted?

**Review by cseufert — COMMENTED** [review]
Looks good

File: tests/errors/errors.php (hunk 1 above)
Comment (czosel) [reply tests/errors/errors.php:55]: I think in this case it's not needed; we're testing the formatting of try/catch already thoroughly, here we just make sure that non-capturing exceptions also work.

Comment (cseufert) [reply tests/errors/errors.php:55]: Makes sense, I think we can merge this then.

**Review by cseufert — APPROVED** [review]

**Pull request merged**